### DISCUSSION TITLE
NHTSA：自駕車過去 10 個月在美國造成了 392 起車禍

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#449] [General]
NHTSA：自駕車過去 10 個月在美國造成了 392 起車禍
自駕車還是免不了出許多車禍，這個技術還需要再繼續發展
https://tw.news.yahoo.com/nhtsa-%E8%87%AA%E9%A7%95%E8%BB%8A%E9%81%8E%E5%8E%BB-10-%E5%80%8B%E6%9C%88%E5%9C%A8%E7%BE%8E%E5%9C%8B%E9%80%A0%E6%88%90%E4%BA%86-392-113029428.html

**Comment by xCk27x** [comment]
https://technews.tw/2020/10/27/tesla-autopilot-stats-2020q3/
目前看來 出事機率其實是相對人工駕車更低的
但是是在有人為監控的情況下

**Comment by brian431** [comment]
現在的自駕技術尚未完全成熟，駕駛人還是必須專注於道路上的情形，不能因為用了自駕就分心。

**Comment by williamyen0623** [comment]
我認爲現在自駕技術還未成熟到能讓駕駛人不用注意路況，但我相信這塊領域是絕對有潛力的，只是需要時間去發展，將來肯定有辦法提升全人類的福祉

**Comment by NcuJerryKao** [comment]
會出車禍要看是人的問題 還是自駕車的
倘若街道上都是自駕車 難道還會出問題嗎

**Comment by Ben6uo** [comment]
但是人要握方向盤 這不可以這說是AI問題

**Comment by skyling0299** [comment]
我覺得AI在道路上行駛最大的問題是不是所有駕駛人都是讓AI行駛
這會導致不可預測性問題

**Comment by BingRong1217** [comment]
希望有一天自駕車能讓車禍不再發生

**Comment by limitx0** [comment]
內文寫｢NHTSA 一共收到了 392 起與自駕或半自駕車『相關』的車禍通報｣
標題寫｢NHTSA：自駕車過去 10 個月在美國『造成』了 392 起車禍｣
感覺新聞有點誤導
我想應該不是所有和自駕車有關的車禍都是自駕車造成的吧

**Comment by larry40305** [comment]
路上有太多不可預測的事，完全不用駕駛的自駕車要實際上路還需要時間

**Comment by ghost** [comment]
自駕車應該是使人生活更加便利的技術，但是人類應該要更謹慎的實用，而不是有了這個就可以直接放著不管，不然縱使科機再怎麼發展，人類也將會自食惡果。

**Comment by bww0401** [comment]
我認為人類很常都會疏忽了，更何況是AI，尤其現在自駕車尚未發展成熟，只能視為輔助，駕駛還是需要當心才行。
